contracts/SSVNetwork.sol

```diff
@@ -127,7 +128,8 @@ contract SSVNetwork is UUPSUpgradeable, Ownable2StepUpgradeable, ISSVNetwork {
 
     function registerOperator(
         bytes calldata publicKey,
-        uint256 fee
+        uint256 fee,
+        address whitelisted
```

Comment:
Doesnt this mean that we will have to deploy new contract?  `address whitelisted` will be new var or is it that you can upgrade and all existing ones will be null?